text/0000-formatted-bliss.md

```diff
@@ -0,0 +1,226 @@
+- Feature Name: `formatted_bliss`
+- Start Date: 2019-05-05
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+1. Introduce a formatting convention (hence: *"the style*")
+   to `rust-lang/rust` (hence: *"the repository*") in a `rustfmt.toml` file.
+
+2. Enforce the style with `src/tools/tidy` in the repository's continuous integration (CI).
+
+3. Add a command `fmt` to rustbuild (`./x.py`).
```

Comment:
Could we make this run as a pre-`./x.py build` hook (or allow configuring this to be the case)? I'd hate to have to wait for everything to rebuild just because a `fmt` ran and messed something up really low down.

Reply:
An opt-in flag to `config.toml` sound good? I wouldn't make it the default as it may throw around code I'm working on and throw me off when developing.

Reply:
sgtm